docs/installation/language.md

```diff
@@ -110,6 +112,21 @@ make -j8
 make install
 ```
 
+Add the install directory path to the MATLAB files to your PATH, and then open MATLAB.
+
+Now in MATLAB, run the following:
+
+```matlab
+loadlibrary(GetFullPath('path/to/helics/install/libhelicsSharedLib.dylib'));
```

Comment:
this again is for mac only,
I will add a section on the mex setup after this is merged